go-controller/pkg/ovn/ovn.go

```diff
@@ -345,7 +345,7 @@ type emptyLBBackendEvent struct {
 }
 
 func newModeEgressIP() modeEgressIP {
-	if config.Gateway.Mode == config.GatewayModeLocal {
+	if config.Gateway.Mode == config.GatewayModeLocal || config.Gateway.Mode == config.GatewayModeHybrid {
```

Comment:
Sigh, this is also wrong for egressIP. Needs to check the *nodes* gateway mode, not the master's configured mode.

Reply:
going to leave this one alone

Reply:
I had no clue different nodes could run in different modes. This is the first time I read that.